newIDE/app/src/AssetStore/InstallAsset.spec.js

```diff
@@ -730,5 +731,43 @@ describe('InstallAsset', () => {
           .toJSArray()
       ).toEqual(['MyBehavior']);
     });
+
+    // TODO EBO Add a test for a custom object that contains another custom objet.
+    // There is 2 cases:
+    // - an event-based object from the same extension (this should already work).
+    // - an event-based object from another extension (this won't work because
+    //   it needs extension dependencies).
+
+    it('install an asset, with an event-based object that is already installed', async () => {
```

Comment:
Should we have a test for "install an asset, with an event-based object that is not already installed" and check that the extension is installed?

Or did I miss it?

Reply:
The tests are unit-testing the function to get extension to install. Probably to avoid having to mock the call to extensions registry.